src/test/java/nl/novi/catsittermanager/services/CatServiceTest.java

```diff
@@ -85,22 +88,51 @@ void testGetCat_shouldFetchCatWithSpecificId() {
         assertEquals(expectedCatDto, actualCatDto);
     }
 
+    @Test
+    void testGetCat_shouldFetchCatWithSpecificId_RecordNotFoundException() {
+        // Arrange
+        Customer owner1 = new Customer();
+        owner1.setUsername("marjet_bosma");
+
+        UUID catId = UUID.randomUUID();
+        Cat cat = new Cat(catId, "Firsa", LocalDate.of(2006, 7, 1), "V", "Europese Korthaar", "Vriendelijke, maar verlegen kat", true, true, "Dierenkliniek Zuilen", "030-2446933", "Thiafeline", "5mg 1dd", owner1);
```

Comment:
This cat is also not needed for this test since it will always return record not found

Reply:
You're right, I only need the catId.